config/jobs/image-pushing/k8s-staging-sig-storage.yaml

```diff
@@ -19,7 +19,7 @@ postsubmits:
       spec:
         serviceAccountName: gcb-builder
         containers:
-          - image: gcr.io/k8s-testimages/image-builder:v20200901-ab141a0
+          - image: gcr.io/k8s-testimages/image-builder:v20200612-cd781f9
```

Comment:
Are you intentionally downgrading the image-builder image?

Reply:
No, that must have crept in while starting from an older base revision. Fixed.